lib/ansible/modules/cloud/ovirt/ovirt_nic.py

```diff
@@ -44,9 +44,11 @@
         description:
             - Logical network to which the VM network interface should use,
               by default Empty network is used if network is not specified.
+            - When not specified and you have only one network and one profile it will automatically select it.
     profile:
         description:
             - Virtual network interface profile to be attached to VM network interface.
+            - When not specified and you have only one network and one profile it will automatically select it.
```

Comment:
When not specified and network has only single profile it will be auto-selected, otherwise you must specify profile.